hail/python/hail/expr/aggregators/aggregators.py

```diff
@@ -238,6 +238,36 @@ def context(self):
             return 'agg'
 
 
+def aggregate_local_array(array, f):
```

Comment:
We had discussed adding a `CollectionExpression.aggregate` method, similar to `fold`. Did you change your mind?

Reply:
nope, forgot over the weekend amid the presentation rush. Thanks for the reminder.